packages/vscode-extension/package.nls.json

```diff
@@ -225,6 +225,12 @@
     "teamstoolkit.handlers.localStateFileNotFound": "State file not found in environment %s. You must run `debug` first to generate related state file.",
     "teamstoolkit.handlers.teamsAppIdNotFound": "Failed to find 'fx-resource-appstudio.teamsAppId' in 'state.%s.json'.",
     "teamstoolkit.handlers.defaultManifestTemplateNotExists": "Manifest template file not found in %s. You can use CLI with your own template file.",
+    "teamstoolkit.handlers.defaultAppPackageNotExists": "App package file not found in %s. You can use CLI with your own app package file.",
```

Comment:
The second sentence does not make sense to me?

Reply:
Now we only support manifest/app package in default path. If user wants to user their own app package with another name or location, they need to use CLL with --app-package-path param.

But this will be refactored in the next release(4/3), then this string will be removed.  So it can be ignored.